stanford_media.install

```diff
@@ -57,3 +57,93 @@ function stanford_media_update_8003() {
   $display->setComponent('field_media_video_embed_field', $display_options);
   $display->save();
 }
+
+/**
+ * Move inline image alt text into a contrib supported attribute.
+ */
+function stanford_media_update_8004() {
```

Comment:
This could be a big update with a whole tonne of MySQL queries. Do you think we need an option to batch it?

Reply:
true dat

Reply:
I added batch updates. I also found that to make sure its all correct, the filter formats need to be altered if they filter html. I added an update hook to do just that. Might want another look before mergeing